src/Acts/CamdramBundle/Service/EmailDispatcher.php

```diff
@@ -44,16 +44,29 @@ public function sendShowCreatedEmail(Show $show, array $owners, array $moderator
         $message = \Swift_Message::newInstance()
             ->setSubject('New show needs authorization on Camdram: '.$show->getName())
             ->setFrom(array($this->from_address => 'camdram.net'))
+            
+            /* HTML */
             ->setTo($toEmails)
             ->setBcc($bccEmails)
             ->setBody(
                 $this->twig->render(
+                    'ActsCamdramBundle:Email:show_created.html.twig',
+                    array(
+                        'owners' => $owners,
+                        'show' => $show,
+                    )
+                ), 'text/html'
+            )
+            
+            /* Plain Text */
+            ->addPart(
+                $this->twig->render(
```

Comment:
This is a second call to "render", whereas the test only expects one call (as before when there was only plain). I presume the fix is to remove the test? (I don't know if you can test for render being called twice?)